### ISSUE TITLE
Bidirectional mask for embedding training w/ megatron

### ISSUE THREAD

**Issue by Lossfull** [modelscope/ms-swift#8164]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new question or discussion topic. / 我已经搜索过现有的 issues，确认这是一个新的问题与讨论。

### Question Description / 问题描述

I want to finetune Qwen/Qwen3-embedding-8B using megatron backend, but i want to modify forward to use bidirectional attention mask, is there a way to do it?
